CrazyCanvas/Source/ECS/Systems/Player/HealthSystem.cpp

```diff
@@ -85,14 +85,15 @@ void HealthSystem::FixedTick(LambdaEngine::Timestamp deltaTime)
 					if (ammoType == EAmmoType::AMMO_TYPE_PAINT)
 					{
 						healthComponent.CurrentHealth -= 10;
-						//LOG_INFO("Player damaged. Health=%d", healthComponent.CurrentHealth);
+						LOG_INFO("Player damaged. Health=%d", healthComponent.CurrentHealth);
 					}
 					else if (ammoType == EAmmoType::AMMO_TYPE_WATER)
 					{
 						healthComponent.CurrentHealth += 10;
-						//LOG_INFO("Player got splashed. Health=%d", healthComponent.CurrentHealth);
+						LOG_INFO("Player got splashed. Health=%d", healthComponent.CurrentHealth);
 					}
 
+					// Set min/max health 
 					if (healthComponent.CurrentHealth <= 0)
 					{
 						LOG_INFO("PLAYER DIED");
```

Comment:
This if-statement can be nested into the if-statement at lines 85-89.

Reply:
Yes